llvm/lib/Transforms/InstCombine/InstCombineNegator.cpp

```diff
@@ -46,12 +46,8 @@
 #include <type_traits>
 #include <utility>
 
-namespace llvm {
```

Comment:
These declarations seems unnecessary as they are already provided by the included headers.